### DISCUSSION TITLE
Send out emails: Restrict product variations

### DISCUSSION THREAD

**Discussion by fefrei** [pretix/pretix#3631] [Ideas]
The generally awesome “Send out emails” feature allows restricting the batch to orders containing a specific product.
Unfortunately, this does not seem to be variation-aware, so it's not possible to send emails to customers with a specific product variation only. This would be helpful, e.g. to send customers details that only apply to a specific variation.

**Comment by raphaelm** [comment]
This has been requested before, also for other places where only products can be selected (questions, …). I understand why it is useful, but I don't think it is worth it. It makes the data model, code and UI significantly more complex, has weird new side-effects (if you create a new variation, you need to update all your scheduled emails where you selected variations).

My vision for products VS variations is that products is **what** you buy and variations are **who** buys, **when** you buy or **how** you buy. In this logic, there are very rare use cases for what you are describing.

**Reply by fefrei** [reply]
Understood. I'd still argue that emails specifically are a use case where separating by variation is especially plausible (example: reminding one customer group to bring proof of eligibility for their discounted variation), and for the once-off emails (which are the more important use case, in my opinion), there's no need to handle newly created variations at all. But of course, I fully understand if this is out of scope 🙂
